api/tests/integration/ArticleTest.php

```diff
@@ -522,9 +528,6 @@ public function shouldGetCommentsForArticleUsingWithNestedHeader()
         $client->api('discussions')->remove($discussion['Discussion']['DiscussionID']);
     }
 
-    /**
-     * @test
-     */
```

Comment:
@artstorm oh right, yea I thought it was a trailing group statement. Actually can you please update the test names to be consistent with the rest of the test suite? i.e. starting with `test`. It doesn't make sense to have two patterns.